DSA/Queue/Queue.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+using namespace std;
+int queue[100], n = 100, front = - 1, rear = - 1;
+void Insert() {
+   int val;
+   if (rear == n - 1)
+   cout<<"Queue Overflow"<<endl;
+   else {
+      if (front == - 1)
+      front = 0;
+      cout<<"Insert the element in queue : "<<endl;
+      cin>>val;
+      rear++;
+      queue[rear] = val;
+   }
+}
+void Delete() {
+   if (front == - 1 || front > rear) {
+      cout<<"Queue Underflow ";
+      return ;
+   } else {
+      cout<<"Element deleted from queue is : "<< queue[front] <<endl;
+      front++;;
+   }
+}
+void Display() {
+   if (front == - 1)
+   cout<<"Queue is empty"<<endl;
+   else {
+      cout<<"Queue elements are : ";
+      for (int i = front; i <= rear; i++)
+      cout<<queue[i]<<" ";
+         cout<<endl;
+   }
+}
+int main() {
+   int ch;
+   cout<<"1) Insert element to queue"<<endl;
+   cout<<"2) Delete element from queue"<<endl;
+   cout<<"3) Display all the elements of queue"<<endl;
```

Comment:
queue has other functions beside insert and delete like front , Please add it.

Reply:
resolved